.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "\U00002728 Feature request"
+description: Suggest an idea for this project
```

Comment:
```suggestion
description: Request a feature
```

Reply:
This is just reordering the words "feature request" and seems redundant. It was also my first thought as well for a description :laughing: